### PR TITLE
Microsite implementation

### PR DESCRIPTION
**Pull request by AntonioMateoGomez**
This PR implements the base `Hood` microsite.
We still need to implement the documentation for both `gradle` and `sbt`. The `sbt` documentation will be part of a different microsite the same way it is beign doing  in [MU](http://higherkindness.io/mu-services/), and the `gradle` documentation will be part of this microsite.

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: Hood
+#-------------------------
+name: Hood
+#-------------------------
+description: Hood is a plugin to compare benchmarks and set the result as a Github status for a Pull Request. Hood is built on Arrow, a Functional companion to Kotlin's Standard Library.
```
Comment (AdrianRaFo) [inline docs/_config.yml]:
```suggestion
description: Hood is a plugin to compare benchmarks and set the result as a Github status and comment on each Pull Request.
```

File: docs/_config.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#-------------------------
+author: 47 Degrees
+keywords: functional-programming, kotlin-library, arrow, monads, monad-transformers, functional-data-structure, kotlin, fp-types, adt, free-monads, tagless-final, mtl, for-comprehension, category-theory
```
Comment (AdrianRaFo) [inline docs/_config.yml]:
```suggestion
keywords: functional-programming, gradle, kotlin, arrow, scala, sbt, plugin, benchmark, comparison, continuous-integration, github
```

**Comment by israelperezglez** [comment]
Good Job @AntonioMateoGomez ;). Thanks for include Bootstrap in the dependency @calvellido

**Comment by calvellido** [comment]
To preview the site locally, from the project root dir:

```bash
bundle install --gemfile docs/Gemfile --path vendor/bundle
```

This, will install website dependencies under `docs/vendor/bundle`. You need to have [ruby >= 2.4.0](https://rvm.io/) and [Bundler >= 2](https://bundler.io/v2.0/guides/bundler_2_upgrade.html) on your system.

```bash
BUNDLE_GEMFILE=./docs/Gemfile bundle exec jekyll serve -s docs -b /hood
```

And this will execute the locally installed Jekyll instance to serve the site.

Then visit:

http://localhost:4000/hood

**Pull request merged**